packages/dev/core/src/PostProcesses/RenderPipeline/Pipelines/ssrRenderingPipeline.ts

```diff
@@ -841,9 +996,9 @@ export class SSRRenderingPipeline extends PostProcessRenderPipeline {
         this._blurPostProcessX = new PostProcess(
             "SSRblurX",
             "screenSpaceReflection2Blur",
-            ["blurQuality", "texelOffsetScale"],
+            ["texelOffsetScale"],
             ["textureSampler"],
-            1,
+            this._useBlur() ? 1 / (this._ssrDownsample + 1) : 1,

```

Comment:
should all the size params be handled in resize as well ?

Reply:
Not sure I understand. Resizing is handled by the post process base class, and textures are recreated with the correct ratio value.